src/ascii/chain/setting/private/v7_36.rs

```diff
@@ -0,0 +1,1122 @@
+    SystemVoltage,
+    /// The [trigger.numactions](https://www.zaber.com/protocol-manual#topic_setting_trigger_numactions) setting.
+    TriggerNumactions,
+    /// The [trigger.numtriggers](https://www.zaber.com/protocol-manual#topic_setting_trigger_numtriggers) setting.
+    TriggerNumtriggers,
+    /// The [user.data.0](https://www.zaber.com/protocol-manual#topic_setting_user_data_0) setting.
+    UserData0,
+    /// The [user.data.1](https://www.zaber.com/protocol-manual#topic_setting_user_data_1) setting.
+    UserData1,
+    /// The [user.data.10](https://www.zaber.com/protocol-manual#topic_setting_user_data_10) setting.
+    UserData10,
+    /// The [user.data.11](https://www.zaber.com/protocol-manual#topic_setting_user_data_11) setting.
+    UserData11,
+    /// The [user.data.12](https://www.zaber.com/protocol-manual#topic_setting_user_data_12) setting.
+    UserData12,
+    /// The [user.data.13](https://www.zaber.com/protocol-manual#topic_setting_user_data_13) setting.
+    UserData13,
+    /// The [user.data.14](https://www.zaber.com/protocol-manual#topic_setting_user_data_14) setting.
+    UserData14,
+    /// The [user.data.15](https://www.zaber.com/protocol-manual#topic_setting_user_data_15) setting.
+    UserData15,
+    /// The [user.data.2](https://www.zaber.com/protocol-manual#topic_setting_user_data_2) setting.
+    UserData2,
+    /// The [user.data.3](https://www.zaber.com/protocol-manual#topic_setting_user_data_3) setting.
+    UserData3,
+    /// The [user.data.4](https://www.zaber.com/protocol-manual#topic_setting_user_data_4) setting.
+    UserData4,
+    /// The [user.data.5](https://www.zaber.com/protocol-manual#topic_setting_user_data_5) setting.
+    UserData5,
+    /// The [user.data.6](https://www.zaber.com/protocol-manual#topic_setting_user_data_6) setting.
+    UserData6,
+    /// The [user.data.7](https://www.zaber.com/protocol-manual#topic_setting_user_data_7) setting.
+    UserData7,
+    /// The [user.data.8](https://www.zaber.com/protocol-manual#topic_setting_user_data_8) setting.
+    UserData8,
+    /// The [user.data.9](https://www.zaber.com/protocol-manual#topic_setting_user_data_9) setting.
+    UserData9,
+    /// The [user.vdata.0](https://www.zaber.com/protocol-manual#topic_setting_user_vdata_0) setting.
+    UserVdata0,
+    /// The [user.vdata.1](https://www.zaber.com/protocol-manual#topic_setting_user_vdata_1) setting.
+    UserVdata1,
+    /// The [user.vdata.2](https://www.zaber.com/protocol-manual#topic_setting_user_vdata_2) setting.
+    UserVdata2,
+    /// The [user.vdata.3](https://www.zaber.com/protocol-manual#topic_setting_user_vdata_3) setting.
+    UserVdata3,
+    /// The [vel](https://www.zaber.com/protocol-manual#topic_setting_vel) setting.
+    Vel,
+    /// The [version](https://www.zaber.com/protocol-manual#topic_setting_version) setting.
+    Version,
+    /// The [version.build](https://www.zaber.com/protocol-manual#topic_setting_version_build) setting.
+    VersionBuild,
```

Comment:
All the types in each module are different, even if they are by definition identical. This means that, for example, `v7_36::Version` and `v7:23::Version` completely incompatible types. Ideally, if the settings are in fact the same, their types would be the same as well.

Reply:
We can deal with that if/when it becomes a problem